api/Blob.json

```diff
@@ -0,0 +1,271 @@
+              },
+              {
+                "version_added": "5",
+                "version_removed": "13",
+                "prefix": "moz",
+                "notes": "The <code>slice()</code> method had initially taken <code>length</code> as the second argument to indicate the number of bytes to include in the new <code>Blob</code>. If you specified values such that <code>start + length</code> exceeded the size of the source <code>Blob</code>, the returned <code>Blob</code> contained data from the start index to the end of the source <code>Blob</code>.&nbsp;That version of the <code>slice()</code> was implemented in <a href='https://hg.mozilla.org/mozilla-central/rev/1b3947ed93c6'>Firefox 4</a>, <a href='http://trac.webkit.org/changeset/55670'>WebKit</a>, and <a href='http://www.opera.com/docs/specs/presto28/file/#blob'>Opera 11.10</a>. However, since that syntax is differed from <a href='/en-US/JavaScript/Reference/Global_Objects/Array/slice'><code>Array.slice()</code></a> and <a href='/en-US/JavaScript/Reference/Global_Objects/String/slice'><code>String.slice()</code></a>, Gecko and WebKit removed support and added support for the new syntax as <code>Blob.slice()</code>/<a href='http://trac.webkit.org/changeset/83873'><code>Blob.webkitSlice</code></a>."
+              }
+            ],
+            "firefox_android": {
+              "version_added": "14"
+            },
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": "12",
+              "notes": "The <code>slice()</code> method had initially taken <code>length</code> as the second argument to indicate the number of bytes to include in the new <code>Blob</code>. If you specified values such that <code>start + length</code> exceeded the size of the source <code>Blob</code>, the returned <code>Blob</code> contained data from the start index to the end of the source <code>Blob</code>.&nbsp;That version of the <code>slice()</code> was implemented in <a href='https://hg.mozilla.org/mozilla-central/rev/1b3947ed93c6'>Firefox 4</a>, <a href='http://trac.webkit.org/changeset/55670'>WebKit</a>, and <a href='http://www.opera.com/docs/specs/presto28/file/#blob'>Opera 11.10</a>. However, since that syntax is differed from <a href='/en-US/JavaScript/Reference/Global_Objects/Array/slice'><code>Array.slice()</code></a> and <a href='/en-US/JavaScript/Reference/Global_Objects/String/slice'><code>String.slice()</code></a>, Gecko and WebKit removed support and added support for the new syntax as <code>Blob.slice()</code>/<a href='http://trac.webkit.org/changeset/83873'><code>Blob.webkitSlice</code></a>."
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": "5.1",
+              "prefix": "webkit",
+              "notes": "This was implemented in <a href='http://trac.webkit.org/changeset/83873'>534.29</a>."
```

Comment:
Remove this note. too. The bug that implemented something is not relevant to anyone anymore.